lib/commands/asm/network/vnetUtil.js

```
@@ -434,7 +434,27 @@ VNetUtil.prototype.getNetworkConfigXml = function(configurationObj) {
     }
   }
 
-  var eleSites = eleDns.insertAfter('VirtualNetworkSites');
+  var eleLocalSites = eleDns.insertAfter('LocalNetworkSites');
+
+  if (configurationObj.VirtualNetworkConfiguration.LocalNetworkSites instanceof Array) {
```

What if 
- configurationObj.VirtualNetworkConfiguration.LocalNetworkSites is not an Array (would you want to throw an error)
- configurationObj.VirtualNetworkConfiguration.LocalNetworkSites is an empty Array?